Maestro/subscriptions.json

```diff
@@ -690,16 +695,39 @@
           "ScriptFileName": "scripts\\Invoke-UpdateDependencies.ps1",
           "Arguments": [
             "-CleanupDocker",
-              "--github-user:dotnet-maestro-bot",
-              "--github-email:dotnet-maestro-bot@microsoft.com",
-              "--github-password:`$Secrets[`'BotAccount-dotnet-maestro-bot-PAT`']",
-              "--branch:dev",
-              "--build-info-url:https://raw.githubusercontent.com/dotnet/versions/master/build-info/dotnet/product/cli/release/2.1/build.xml"
+            "--github-user:dotnet-maestro-bot",
+            "--github-email:dotnet-maestro-bot@microsoft.com",
+            "--github-password:`$Secrets[`'BotAccount-dotnet-maestro-bot-PAT`']",
+            "--branch:dev",
+            "--build-info-url:https://raw.githubusercontent.com/dotnet/versions/master/build-info/dotnet/product/cli/release/2.1/build.xml"
           ]
         },
         "vsoSourceBranch": "dev"
       }
     },
+    // Update dependencies in the aspnetcore repo's dev branch
+    {
+      "triggerPaths": [
+        "https://github.com/dotnet/versions/blob/master/build-info/dotnet/product/cli/master/packages.semaphore"
+      ],
+      "action": "aspnet-dependency-update",
+      "delay": "00:05:00",
+      "actionArguments": {
+        "vsoSourceBranch": "master",
+        "vsoBuildParameters": {
+          "ScriptFileName": "scripts\\UpdateDependencies.ps1",
+          "Arguments": [
+            "-BuildXml",
+            "https://raw.githubusercontent.com/dotnet/versions/master/build-info/dotnet/product/cli/master/build.xml",
+            "-ConfigVars '",
+            "--GithubUsername dotnet-maestro-bot",
```

Comment:
I recommending indenting the args within the `'`, but since it's your subscription I'm fine if you don't want to here.